DakeAndDevileCorps/Screens/MapHome/VC/MainMapViewController.swift

```diff
@@ -9,25 +9,57 @@ import MapKit
 import UIKit
 
 class MainMapViewController: UIViewController {
-
+    
+    // MARK: - subViews
     private let searchBarView: SearchBarView = {
         $0.translatesAutoresizingMaskIntoConstraints = false
         $0.entryPoint = .map
         return $0
     }(SearchBarView())
     
+    private let categoryView: CategoryView = {
+        let categoryView = CategoryView(entryPoint: .map)
+        categoryView.translatesAutoresizingMaskIntoConstraints = false
+        return categoryView
+    }()
+    
     @IBOutlet weak var mapView: MKMapView!
     
+    // MARK: - properties
+    var shops: [MKAnnotation] = [
```

Comment:
p5;

```suggestion
    var shops: [ShopInfo] = [
```
이렇게 쓰면 118번 라인에서 guard 문으로 타입 캐스팅해주지 않아도 될 거 같아요.
ShopInfo 자체가 MKAnnotation를 상속받도록 구현해주셨으니깐 MKAnnotation값이 쓰여야 하는 부분에서도 잘 들어갈 거 같구요!

Reply:
오 샤프한 지적 감사합니다! [9db9673](https://github.com/DeveloperAcademy-POSTECH/MC3-Team4-DakeAndDevileCorps/pull/69/commits/9db9673c1c9e2225e68b7374632619e6626a4056)